plugins/discovery-ec2/src/main/java/org/elasticsearch/plugin/discovery/ec2/DiscoveryEc2Plugin.java

```diff
@@ -37,49 +33,32 @@
 /**
  *
  */
-public class CloudAwsPlugin extends Plugin {
-
-    private final Settings settings;
-
-    public CloudAwsPlugin(Settings settings) {
-        this.settings = settings;
-    }
+public class DiscoveryEc2Plugin extends Plugin {
 
     @Override
     public String name() {
-        return "cloud-aws";
+        return "discovery-ec2";
     }
 
     @Override
     public String description() {
-        return "Cloud AWS Plugin";
+        return "Discovery EC2 Plugin";
```

Comment:
While I think the name in maven should keep with prepending the plugin type, it makes more sense gramatically to say "EC2 Discovery Plugin" in the description (and this is what I did for multicast).